components/server/src/github/github-context-parser.ts

```diff
@@ -145,15 +145,15 @@ export class GithubContextParser extends AbstractContextParser implements IConte
                 );
             }
             const defaultBranch = result.data.repository.defaultBranchRef;
-            const ref = (defaultBranch && defaultBranch.name) || undefined;
+            const ref = (defaultBranch && defaultBranch?.name) || undefined;
```

Comment:
why adding the `?` if we check the existence of `defaultbranch` before?

Reply:
fixed